packages/botonic-react/src/constants.js

```diff
@@ -125,6 +125,7 @@ export const WEBCHAT = {
     // TriggerButton
     customTrigger: 'triggerButton.custom',
     triggerButtonImage: 'triggerButton.image',
+    triggerButtonNotifications: 'triggerButton.notifications',
```

Comment:
Change: This should be changed to:
```suggestion
    triggerButtonNotificationsEnabled: 'triggerButton.notificationsEnabled',
```